site2/website/versioned_docs/version-2.6.2/administration-zk-bk.md

```diff
@@ -197,44 +230,43 @@ $ bin/bookkeeper shell bookiesanity
 
 This command creates a new ledger on the local bookie, writes a few entries, reads them back and finally deletes the ledger.
 
-### Hardware considerations
-
-Bookie hosts are responsible for storing message data on disk. In order for bookies to provide optimal performance, ensuring that the bookies have a suitable hardware configuration is essential. You can choose two key dimensions to bookie hardware capacity:
-
-* Disk I/O capacity read/write
-* Storage capacity
+### Decommissioning bookies cleanly
 
-Message entries written to bookies are always synced to disk before returning an acknowledgement to the Pulsar broker. To ensure low write latency, BookKeeper is designed to use multiple devices:
+When you want to decommission a bookie, the following process is useful to follow in order to verify if the decommissioning was safely done.
 
-* A **journal** to ensure durability. For sequential writes, having fast [fsync](https://linux.die.net/man/2/fsync) operations on bookie hosts is critical. Typically, small and fast [solid-state drives](https://en.wikipedia.org/wiki/Solid-state_drive) (SSDs) should suffice, or [hard disk drives](https://en.wikipedia.org/wiki/Hard_disk_drive) (HDDs) with a [RAID](https://en.wikipedia.org/wiki/RAID)s controller and a battery-backed write cache. Both solutions can reach fsync latency of ~0.4 ms.
-* A **ledger storage device** is where data is stored until all consumers have acknowledged the message. Writes happen in the background, so write I/O is not a big concern. Reads happen sequentially most of the time and the backlog is drained only in case of consumer drain. To store large amounts of data, a typical configuration involves multiple HDDs with a RAID controller.
+#### Before you decommission
+1. Ensure state of your cluster can support the decommissioning of the target bookie.
+Check if `EnsembleSize >= Write Quorum >= Ack Quorum` stays true with one less bookie
 
+2. Ensure target bookie shows up in the `listbookies` command.
 
+3. Ensure that there is no other process ongoing (upgrade etc).
 
-### Configure BookKeeper
+#### Process of Decommissioning
+1. Log on to the bookie node, check if there are underreplicated ledgers.
 
-you can find configurable parameters for BookKeeper bookies in the [`conf/bookkeeper.conf`](reference-configuration.md#bookkeeper) file.
+If there are, the decommission command will force them to be replicated.
+`$ bin/bookkeeper shell listunderreplicated`
 
-Minimum configuration changes required in `conf/bookkeeper.conf` are:
+2. Stop the bookie by killing the bookie process. Make sure there are no liveness / readiness probes setup for the bookies to spin them back up if you are deployed in a kubernetes environment.
```

Comment:
```suggestion
2. Stop the bookie by killing the bookie process. Make sure there are no liveness/readiness probes setup for the bookies to spin them back up if you are deployed in a kubernetes environment.
```
 "if you are deployed in a kubernetes environment", this sentence does not use correct grammatical structure, please improve it.